### ISSUE TITLE
Move help source files to where people can see them

### ISSUE THREAD

**Issue by vadosnaprimer** [TASVideos/fceux#274]
Currently here:
https://github.com/TASVideos/fceux/tree/master/vc/Help

Since it's hidden so well, everyone was editing HTML files directly instead, which resulted in all their edits being wiped away upon proper update, which consists of editing the HND files and generating output in CHM and HTML format, as the readme explains.

@bbbradsmith would you like to put some info about your contributions into 2.3.0 whatsnew section (in https://github.com/TASVideos/fceux/blob/master/vc/Help/fceux.hnd)? I [resurrected](https://github.com/TASVideos/fceux/commit/a4b655e64917e17c7dc4a090fafa2906ed5e5b3a) the documentation edits, but I haven't documented everything you're actually coded, since there's a ton.

**Comment by bbbradsmith** [comment]
Hmm. I can see why I never realized about the HND being source for those. That's a format I've never heard of, and being a binary file type we can't search it, and we can't see what's changed if anyone edits it. Also, as far as I can tell free version 3 of HelpNDoc is no longer available, and it looks like any attempt to edit the HND with the newer version is going to do some sort of "upgrade" to it, so I'm very hesitant to try editing them myself in the current state.

Here's a list of changes I contributed since 2.2.3 (July 2016):

* Lua memory write callbacks can now utilize memory written https://sourceforge.net/p/fceultra/code/3266/
* Read/write value breakpoint conditions https://sourceforge.net/p/fceultra/code/3268/
* Hex editor can now view OAM and dump to file, other minor improvements https://sourceforge.net/p/fceultra/code/3271/
* RAM initialization options (default, random, all 0, all 1) https://sourceforge.net/p/fceultra/code/3275/
* Relative-position mouse support for better fullscreen mouse capture https://sourceforge.net/p/fceultra/code/3278/
* Better colours available to lua script overlays https://sourceforge.net/p/fceultra/code/3280/
* Fill colour option for viewing with background layer disabled https://sourceforge.net/p/fceultra/code/3282/
* Fix silly quit message randomizer https://sourceforge.net/p/fceultra/code/3283/
* Fix corruption of PPU reset while debugging https://sourceforge.net/p/fceultra/code/3284/
* Mapper 111 Cheapocabra support https://sourceforge.net/p/fceultra/code/3287/
* Attribute mode for nametable viewer https://sourceforge.net/p/fceultra/code/3301/
* Greyscale palettes for PPU/nametable viewers https://sourceforge.net/p/fceultra/code/3304/
* Fix Windows UI font rescaling issue https://sourceforge.net/p/fceultra/code/3314/
* Fix transparent sprite colour display in PPU viewer https://sourceforge.net/p/fceultra/code/3331/
* Fix NTSC filter null pointer crash when closing (submitted by Dustin Long) https://sourceforge.net/p/fceultra/code/3374/
* 4-screen support for Mapper 30 (UNROM512) 9b6ff8fe490cae56f6a3bc24e0ac19e47b5eaa41
* Fixed emphasis support for NTSC filter e8c61d1b2367b6df2c54d8073ff0c73c2f50abc4
* WRAM now affected by memory initialization options 5be92d3ee50fcdc04ec4d727cef5201fa8fba378
* Fix crash for lua script that fails to load 349aed4aaf0576101d034add1ffc492b2fa7354c
* Added tokumaru's helpful sprite debugging script (luaScript/Sprites.lua) bc41bc80ee6d5f10f359f2f5d27670a5a22327fd
* Fix mouse buttons conflicting with each other 01c3f40f1d1deb0050452ce79c5ce64df5040c57 20f0783a96851273c510a762886f2db2ad88ae70
* VRC7 patch set finalized with correct values dumped by NukeYKT 20f0783a96851273c510a762886f2db2ad88ae70
* Windows 64-bit build 72120b0bb9fbe268eb6839ce1400d4a75df31ea0
* Fix debugging and rendering of mirrored palette entries 5c8421d410573f11370c545ae13235f116bf5137
* Fix debugger filtering special symbols fbabd180ebf4c1a42e544a9bd7628ec00d20951c
* MMC5 large WRAM support 88d7f392a98bcba02116ca34e25c83da07608de4
* Zapper control lua support c4ee998148ee6e71d7888b2b1ea7d2d040ed9dd8
* Virtual Boy controller support b22753762fbbbbb93559e83b2692a31082519d06
* Fix memory corruption vulnerability with CHR <1k e8597862f1f35b8c5d4f8570f256750395a7716b
* Improve error message for invalid ROM file 2cdb283a64413fc0aee283433cc02092cce4dc69

**Comment by bbbradsmith** [comment]
It looks like my documentation edits are not in the HND, so I'll have to review all of that as well.

**Comment by vadosnaprimer** [comment]
There's a dedicated toolbox button for searching, Ctrl+F does nothing.

**Comment by bbbradsmith** [comment]
Ah thanks. I had looked but missed it. It's in the "Write" ribbon page, all the way on the right side, as "Find and Replace".

**Comment by bbbradsmith** [comment]
Missing documentation changes that I was involved with:

* Lua memory callback has third parameter now https://sourceforge.net/p/fceultra/code/3267/
* Additions to debugger breakpoint conditions https://sourceforge.net/p/fceultra/code/3269/
* OAM memory view https://sourceforge.net/p/fceultra/code/3272/
* Greyscale palette https://sourceforge.net/p/fceultra/code/3306/
* Explanation of Sprites.lua bc41bc80ee6d5f10f359f2f5d27670a5a22327fd
* Zapper lua control: ce16e4741073c91225e0c85a63aa2b2115b8a87a + e79bbc0b1c0e1e120cea907f7271b238ddbede62

I can see other changes in these two places, so I'm not sure if I was the only person editing the HTML directly by mistake? I haven't taken the time to look at the rest of these:

* https://sourceforge.net/p/fceultra/code/3396/log/?path=/web
* https://github.com/TASVideos/fceux/commits/master/web/help

**Comment by bbbradsmith** [comment]
Okay, after getting the HND editing working I reviewed it for everything I listed above. PR above for the missing stuff.

**Comment by ClusterM** [comment]
Too bad than HND is a binary format. Maybe we should find some alternative?

**Comment by vadosnaprimer** [comment]
I agree but someone needs to find an alternative that can generate both CHM and HTML while using text files as a source.
